lib-dmtf/model/storage.go

```diff
@@ -146,3 +146,183 @@ type NVMeSMARTCriticalWarnings struct {
 	PowerBackupFailed        bool `json:"PowerBackupFailed,omitempty"`
 	SpareCapacityWornOut     bool `json:"SpareCapacityWornOut,omitempty"`
 }
+
+// Drive schema represents a single physical drive for a system
+type Drive struct {
+	OdataContext                  string             `json:"@odata.context,omitempty"`
+	Etag                          string             `json:"@odata.etag,omitempty"`
+	OdataID                       string             `json:"@odata.id"`
```

Comment:
use camel case for this, type etc consistenly

Reply:
updated